doc/man3/PKCS12_add_cert.pod

```diff
@@ -30,33 +35,40 @@ safeBags. If I<key_nid> is not -1 then the key is encrypted with the supplied
 algorithm, using I<pass> as the passphrase and I<iter> as the iteration count. If
 I<iter> is zero then a default value for iteration count of 2048 is used.
 
+PKCS12_add_key_ex() is identical to the above but allows for a library context I<ctx>
```

Comment:
...is identical to PKCS_add_key() but allows...

It's safer to specify the function by name.